sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/file/datalake/perf/ReadFileDatalakeTest.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.perf;
+
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.storage.file.datalake.DataLakeFileAsyncClient;
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.azure.storage.file.share.ShareFileAsyncClient;
+import com.azure.storage.file.share.ShareFileClient;
+import com.azure.storage.file.datalake.perf.core.DirectoryTest;
+import reactor.core.publisher.Mono;
+
+import java.io.OutputStream;
+import java.util.UUID;
+
+import static com.azure.perf.test.core.TestDataCreationHelper.createRandomByteBufferFlux;
+
+public class ReadFileDatalakeTest extends DirectoryTest<PerfStressOptions> {
+    private static final int BUFFER_SIZE = 16 * 1024 * 1024;
+    private static final OutputStream DEV_NULL = new NullOutputStream();
+    private static final String FILE_NAME = "perfstress-filev11-" + UUID.randomUUID().toString();
+
+    protected final DataLakeFileClient dataLakeFileClient;
+    protected final DataLakeFileAsyncClient dataLakeFileAsyncClient;
+
+    private final byte[] buffer = new byte[BUFFER_SIZE];
+
+    public ReadFileDatalakeTest(PerfStressOptions options) {
+        super(options);
+        dataLakeFileClient = dataLakeDirectoryClient.getFileClient(FILE_NAME);
+        dataLakeFileAsyncClient = dataLakeDirectoryAsyncClient.getFileAsyncClient(FILE_NAME);
+    }
+
+    // Required resource setup goes here, upload the file to be downloaded during tests.
+    public Mono<Void> globalSetupAsync() {
+        return super.globalSetupAsync()
+            .then(dataLakeFileAsyncClient.create())
+            .then(dataLakeFileAsyncClient.upload(createRandomByteBufferFlux(options.getSize()), null, true))
+            .then();
+    }
+
+    // Perform the API call to be tested here
+    @Override
+    public void run() {
+        dataLakeFileClient.read(DEV_NULL);
+    }
+
+    static class NullOutputStream extends OutputStream {
```

Comment:
Can we move this into the base perf framework, this is likely something we'll want across SDKs.